docs/setup/plugins/initiatives.md

```diff
@@ -1,5 +1,165 @@
 ---
 title: Initiatives
+description: Setting up the Initiatives plugin.
 ---
 
+The SourceCred Initiatives plugin adds contributions to the graph using data
+from JSON files following the [Initiatives concept].
+
+## Status and Caveats
+
+[This plugin][plugin source] is currently in Beta.
+
+Because it's recently been added (April 2020), we're still gaining experience
+with this plugin in practice. Any current guides and recommendations may change
+as a result.
+
+One caveat is that an Initiative mixes objective data (who did what) with
+subjective data (how valuable was that). In SourceCred it's a feature that we
+can retroactively update those subjective weights. The objective data helps have
+constructive discussions about, "do we have our facts straight?" and helps to
+remember what made an Initiative valuable in a future discussion.
+
+Try to avoid shaping the objective data to fit your subjective needs. Such as
+leaving out a contribution because you feel the weight should be lower than some
+other contributions.
+
+The biggest shortcoming of this plugin currently is around _inclusivity_.
+
+Creating / updating Initiatives is rather technical, as you need to hand write a
+JSON file and understand how the plugin works to do so. It also requires a
+review process because this plugin allows almost arbitrary additions to the
+Graph, which adds another barrier to being recognized. Validating whether
+everything has been correctly attributed is also not easy in the current UI,
+adding another technical barrier.
+
+We're aware of this issue and plan on improving this with more accessible tools.
+In the meantime if you're using this plugin, it's important to be aware of these
+barriers and support your community in having access to fair recognition.
```

Comment:
I feel like this section is content that even non technical people would want to know, it might fit better in the 'guides/initiatives' doc with a link to that section in this technical guide.

Reply:
Agreed. I sense that a lot of content we're creating is going to get shuffled around a bit before the overall docs settle, but the text is solid.

Reply:
Good point. I was thinking it fits here, because if you're reading the setup guide you're probably the person who's working on getting it adopted and it would be good if they're aware of the limitations. Plus they'd be the technical audience that would be in a good place to improve that inclusivity.

But it definitely doesn't only concern the techies who roll out the plugin. I'm not super clear on the guides audience/scope/tone. Since there isn't precedent there yet.

Some of the ideas on roam and the TODO pages are at the level of: how do I start earning cred on platform X? Going from there to, what are the caveats using a particular plugin, seems like a more advanced topic. It feels like, how would I get my contributions included in an initiative, is the sort of question we're trying to answer in that category.

Anyway, for now I'll leave it here so it exists. Let's discuss moving it in a follow-up.

Reply:
Tracking as a refinement here: https://github.com/sourcecred/docs/issues/65